default.nix

```diff
@@ -249,8 +243,10 @@ in rec {
           (pkgs.path + /nixos/modules/security/acme.nix)
         ];
         nixpkgs.overlays = [
-          (self: super: {
-            simp_le = nixos1909.simp_le;
+          (self: super: let
+            nixos1909 = import (hackGet ./dep/nixpkgs-19.09) {};
```

Comment:
@ericson2314 Shouldn't this just be `import ./dep/nixpkgs-19.09 {}`? hackGet means the default.nix is never read.